### PR TITLE
implement -number-of-tasks cli arg

### PR DESCRIPTION
**Pull request by sherzberg**
I needed a way to run multiple instances of the same task at once so this PR helps with that.

### CODE REVIEW

**Comment by lox** [comment]
Thanks! How about we go with `--count` to match the `Count` param on `RunTaskInput`?

**Comment by sherzberg** [comment]
Thanks for the quick response! Count sounds great. Just updated my pr.

**Comment by lox** [comment]
Thanks for the contribution!

File: main.go [hunk 1]
```diff
@@ -92,6 +97,7 @@ func main() {
 		r.SecurityGroups = ctx.StringSlice("security-group")
 		r.Subnets = ctx.StringSlice("subnet")
 		r.Environment = ctx.StringSlice("env")
+		r.Count = ctx.Int64("number-of-tasks")
```
Comment (sherzberg) [inline main.go]: @lox sorry, but I missed this! New PR coming :(

Comment (lox) [reply main.go]: Eeep I should have caught that too, sorry! Too early in the morning here :D

**Comment by lox** [comment]
Released in https://github.com/buildkite/ecs-run-task/releases/tag/v1.3.0, thanks for taking the time to put in a PR @sherzberg, we welcome any future changes or issues you encounter!

**Pull request merged**